app/components/shared/main-navigation.html

```diff
@@ -1,5 +1,8 @@
 <ul>
     <li>
+        <a href="#" [routerLink]="['/accounts', selectedId]" routerLinkActive="active"><i class="fa fa-th fa-lg"></i><span>Dash</span></a>
```

Comment:
there might be a problem here. In the image it doesn't show the dashboard as active